week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -13,12 +13,24 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+  // b is not defined.
   2. Remove the line that throws the error.
+  //Done!
   3. What is printed to the console?
-  4. How many times is "f1" called?
+  // 2 6 4 9 6 13 8
+  when i = 0  a =7   f2 is called and gives 4
+  when i = 1  a = 8  f1 is called and gives 9
+  when i = 2  a = 9  f2 is called and gives 6
+  when i = 3  a =10  f1 is called and gives 13
+  when i = 4  a =11  f2 is called and gives 8
```

Comment:
The loop is a really good idea
putting the values of the variables will be more clear
 x = 2;
      a = 6;